content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,125 @@
+
+## Habt Ihr Vorschläge für Beiträge?  
+
+<img class="offset-image offset-image-right" src="/card-icons/code.svg" height="150px" width="190px" style="height: 150px; margin-right: -100px; margin-top: -50px;" alt="">
+
+Gerade wenn Du neu bei Datenanfragen.de bist, weißt Du wahrscheinlich gar nicht, wo Du anfangen kannst. Keine Sorge: Es gibt bei uns zahlreiche Bereiche, in denen Du mitmachen kannst – ganz unabhängig davon, ob Du Erfahrung mit Programmierung und Datenschutz hast oder nicht. Zum leichteren Einstieg haben wir Dir hier einmal ein paar Vorschläge zusammengestellt. Viele weitere Aufgaben findest Du in den Issues der jeweiligen Repositories. Wir freuen uns natürlich auch, wenn Du eigenen Ideen hast, was Du am Projekt verbessern könntest.
+
+* **Rückmeldungen zu Anfragen an Unternehmen**  
+  Für DSGVO-Anfragen muss das Unternehmen Dich eindeutig identifizieren können. Die Daten, die dafür nötig sind, unterscheiden sich von Unternehmen zu Unternehmen. Deshalb sammeln wir diese Information in unserer Unternehmensdatenbank.
+
+  Diese Daten fehlen uns noch für viele Unternehmen, deshalb nutzen wir für diese im Generator die Standardfelder (Name, E-Mail und Adresse). Diese passen aber leider nicht immer, manche Unternehmen fordern noch mehr, wie z.B. spezielle Nutzernamen oder sind mit weniger zufrieden.
+  Du kannst uns helfen die Einträge entsprechend zu erweitern. In [diesem GitHub-Issue](https://github.com/datenanfragen/data/issues/720) erkären wir genau, wie Du durch Anfragen an die Unternehmen die entsprechenden Daten nachtragen kannst. Als netter Nebeneffekt kannst Du dadurch auch erfahren, was das Unternehmen so über dich weiß und praktische Erfahrung mit der DSGVO sammeln.
+
+* **Leichte oder einfache Sprache**
+  Kennst Du dich mit leichter oder einfacher Sprache aus und hast Lust für uns einen generellen Überblick oder Artikel zu schreiben? Wir möchten Datenschutz und die DSGVO-Rechte jeder Person leicht zugänglich machen und würden uns freuen, in Zukunft auch Inhalte in leichter oder einfacher Sprache anzubieten. Schreibe uns dafür gerne eine E-Mail an <hacktoberfest@datenanfragen.de>.
+
+* **Schreiben und Übersetzen von Artikeln**  
+  In unserem {{< link slug="blog" text="Blog" >}} haben wir Artikel zu vielen Datenschutzthemen. Wir wollen gerne mehr Artikel veröffentlichen und wollen dort auch gerne weiterhin ein großes Spektrum abdecken, so haben wir bislang z.B. Erklärungen der DSGVO-Rechte, Anleitungen, Erfahrungsberichte und Recherchen. Reiche neue Beiträge bitte im [website-Repository](https://github.com/datenanfragen/website) ein.
+
+  Eine weitere wichtige Aufgabe ist das Übersetzen dieser Artikel in die anderen Sprachen, die wir unterstützen. Auch dafür ist das [website-Repository](https://github.com/datenanfragen/website) der richtige Ort.
```

Comment:
We should mention the relevant issues and which languages they are for (also in the English version, of course):

* https://github.com/datenanfragen/website/issues/489
* https://github.com/datenanfragen/website/issues/457
* https://github.com/datenanfragen/website/issues/456
* https://github.com/datenanfragen/website/issues/455
* https://github.com/datenanfragen/website/issues/695
* https://github.com/datenanfragen/website/issues/696
* https://github.com/datenanfragen/website/issues/697